app/src/main/java/ai/brokk/cli/BrokkCli.java

```diff
@@ -550,6 +581,182 @@ public Integer call() throws Exception {
                     }
                     result = InstructionsPanel.executeAskCommand(cm, codeModel, askPrompt);
                     context = scope.append(result);
+                } else if (reviewPrompt != null) {
```

Comment:
This procedure is pretty long. I don't want to slow us down too much, but I wonder if it would be worth it to parameterize and refactor out. That would also avoid the <invoke CLI> then read file style of integration -- we just invoke this method returning the JSON in-memory.